test/indexer/test_indexer.py

```diff
@@ -351,7 +351,7 @@ def test_zero_tallies(self):
         for doc_id in doc_ids:
             message_re = re.compile(fr'INFO:elasticsearch:'
                                     fr'Got 201 response after [^ ]+ from PUT to '
-                                    fr'.*_aggregate/_doc/{doc_id}.*')
+                                    fr'.*_aggregate/_create/{doc_id}.*')
```

Comment:
This looks odd to me. `_doc` is what ES calls a `doctype`, the default one. They are deprecated, so maybe this is ok.